src/components/tab-bar/partial-styles/tab-bar-scroller.scss

```diff
@@ -0,0 +1,86 @@
+.scroll-fade,
+.scroll-button {
+    position: absolute;
+    transition: {
+        property: transform;
+        duration: 0.3s;
+        delay: 0.5s;
+        timing-function: ease-out;
+    }
+}
+
+.scroll-fade {
+    top: 0;
+    height: 100%;
+    width: pxToRem($tab-scroller-fade-width);
+    pointer-events: none;
+
+    &.left {
+        transform: translate3d(pxToRem(0 - $tab-scroller-fade-width), 0, 0);
+        left: 0;
+        background: linear-gradient(
+            270deg,
+            rgba($tab-background-color, 0) 0%,
+            rgba($tab-background-color, 0.8) 40%,
+            rgba($tab-background-color, 0.8) 100%
+        );
+    }
+
+    &.right {
+        transform: translate3d(pxToRem($tab-scroller-fade-width), 0, 0);
+        right: 0;
+        background: linear-gradient(
+            90deg,
+            rgba($tab-background-color, 0) 0%,
+            rgba($tab-background-color, 0.8) 40%,
+            rgba($tab-background-color, 0.8) 100%
+        );
+    }
+}
+
+.scroll-button {
+    --icon-background-color: rgb(var(--contrast-100));
+    top: pxToRem(4);
+
+    &.left {
+        transform: translate3d(pxToRem(0 - $tab-scroller-fade-width), 0, 0);
+        left: pxToRem(4);
+    }
+
+    &.right {
+        transform: translate3d(pxToRem($tab-scroller-fade-width), 0, 0);
+        right: pxToRem(4);
+    }
+
+    &:hover {
+        transform: translate3d(0, 0, 0);
+    }
+}
+
+.mdc-tab-scroller {
+    position: relative;
+    overflow: hidden;
+
+    &.scroll-left .scroll-fade.left,
+    &.scroll-left .scroll-button.left,
+    &.scroll-right .scroll-fade.right,
+    &.scroll-right .scroll-button.right {
+        transform: translate3d(0, 0, 0);
+    }
+
+    &.scroll-left:not(.scroll-right) .scroll-button.right,
+    &.scroll-right:not(.scroll-left) .scroll-button.left {
+        opacity: 0.5;
+    }
+}
+
+.mdc-tab-scroller__scroll-area--scroll {
+    scrollbar-width: none; // This hides the scrollbars appearing under the tab bar in Firefox
+    -ms-overflow-style: none; // Same as above for IE 11
```

Comment:
have you actually tried this in IE? 🙈 😁

Reply:
I think I did actually on browser stack :)

Reply:
We don't support IE, so there's no reason to add this. Also, our build-process should add any necessary prefixes automatically. I'm not sure that works if the properties aren't the same, but I think we should default to assuming manual adding of vendor-prefixed properties is not necessary, and only add them manually after confirming that it is through manual testing in the browser in question.

Reply:
if you want me to change it or test it again, please make a new issue for it 👊😈
😁

Reply:
This was added in a previous PR, and just moved from one file to another in this one, so let's just leave it in there for now. I'm sure Kia will get annoyed with it and remove it in some future PR 😆